### DISCUSSION TITLE
Set up connection via ESPHome BTProxy

### DISCUSSION THREAD

**Discussion by orgebar** [dbuezas/eq3btsmart#70] [Q&A]
Hi,
Is there any user guide for newbies on how to set up the connection for the Thermostats using BTProxy? I managed to set up an ESP32 with ESPHome, but I'm totally lost when looking at the ESPHome User Guide and how to adapt the steps to use/set up the connection to the thermostats using eq3btsmart integration.
Any help is welcome :)! Thanks!

**Comment by dbuezas** [comment]
Hey there 👋 
You need to:
* Make sure the esphome integration is up and running in Home assistant (follow their tutorial)
* The btproxy device is picked up by home assistant (if it's connected to the network, Home Assistant should auto discover it)
* ensure this custom integration is installed and working (e.g copy files, restart HA, and confirm `dbuezas eq3` is in the list when you click "add device" in the integrations page of Home Assistant)
* At this point, the eq3 should be autodiscovered by home assistant, just follow the ui flow

And it should work. I find usb Bluetooth dongles to work better, but other people report that btproxy works fine.
